javascript-carousel/main.js

```diff
@@ -0,0 +1,91 @@
+
+var $columnFull = document.querySelector('.column-full');
+var $faCircle = document.querySelectorAll('.fa-circle');
+var $faChevronLeft = document.querySelector('.fa-chevron-left');
+var $faChevronRight = document.querySelector('.fa-chevron-right');
+var $img = document.querySelector('img');
+
+var index = 0;
+var intervalID = null;
+
+function updateImage() {
+  if (index === 0) {
+    $img.setAttribute('src', 'images/001.png');
+    $img.setAttribute('alt', 'Bulbasaur');
+  } else if (index === 1) {
+    $img.setAttribute('src', 'images/004.png');
+    $img.setAttribute('alt', 'Charizard');
+  } else if (index === 2) {
+    $faCircle.className = ('far fa-circle spacing');
+    $faCircle[2].className = ('fas fa-circle spacing');
+    $img.setAttribute('src', 'images/007.png');
+    $img.setAttribute('alt', 'Squirtle');
+  } else if (index === 3) {
+    $faCircle[2].className = ('far fa-circle spacing');
+    $faCircle[3].className = ('fas fa-circle spacing');
+    $faCircle[4].className = ('far fa-circle spacing');
+    $img.setAttribute('src', 'images/025.png');
+    $img.setAttribute('alt', 'Pikachu');
+  } else if (index === 4) {
+    $faCircle[3].className = ('far fa-circle spacing');
+    $faCircle[4].className = ('fas fa-circle spacing');
+    $faCircle[0].className = ('far fa-circle spacing');
+    $img.setAttribute('src', 'images/039.png');
+    $img.setAttribute('alt', 'Jiggly Puff');
+  }
```

Comment:
If you make an array of the list of image URLs in order, you can do this with a loop rather than needing a conditional block for each image